backend/Utils/EmailUtils.ts

```diff
@@ -52,3 +80,13 @@ export async function sendEmail({ to, subject }) {
     throw Error(`Call to Gmail API failed with status ${result.status}`);
   }
 }
+
+function makeJwt(email: string): string {
```

Comment:
This shouldn't be in email utils since JWTs are not really related to emails. We could create a JWTUtils or just put it right in ResetPassword.ts. Let's create a JWTUtils.ts file and put it in there. The payload should be passed in rather than the email address, since many JWTs will be different than just the email address